src/Stryker.Core/Stryker.Core.UnitTest/Options/Inputs/IgnoreLinqExpressionInputTests.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Shouldly;
+using Stryker.Core.Exceptions;
+using Stryker.Core.Mutators;
+using Stryker.Core.Options.Inputs;
+using Xunit;
+
+namespace Stryker.Core.UnitTest.Options.Inputs
+{
+    public class IgnoreLinqExpressionInputTests : TestBase
+    {
+        private IEnumerable<LinqExpression> AllLinqExpressions { get; } = Enum.GetValues(typeof(LinqExpression))
+                    .Cast<LinqExpression>()
+                    .Where(w => w != LinqExpression.None);
+
+        [Fact]
+        public void ShouldHaveHelpText()
+        {
+            var target = new IgnoreLinqExpressionInput();
+            target.HelpText.ShouldBe(@"The given linq expression will be excluded for this mutation testrun.
+    This argument takes a json array as value. Example: ['linq.FirstOrDefault', 'linq.First'] | default: []");
+        }
+
+        [Fact]
+        public void ShouldReturnEmptyWithNonLinqOptions()
+        {
+            var target = new IgnoreLinqExpressionInput { SuppliedInput = new[] { "gibberish" } };
+            var result = target.Validate();
+            result.ShouldBeEmpty();
+        }
+
+
+        [Theory]
+        [InlineData("linq.nothing")]
+        [InlineData("linq.test")]
+        public void ShouldValidateExcludedLinqExpression(string method)
+        {
+            var target = new IgnoreLinqExpressionInput
+            {
+                SuppliedInput = new[] { method }
+            };
+
+            var ex = Should.Throw<InputException>(() => target.Validate());
+
+            ex.Message.ShouldBe($"Invalid excluded linq expression ({method.Split(".").Last()}). The excluded linq expression options are [{string.Join(", ", AllLinqExpressions)}]");
+        }
+
+        [Fact]
+        public void ShouldHaveDefault()
+        {
+            var target = new IgnoreLinqExpressionInput { SuppliedInput = new string[] { } };
+
+            var result = target.Validate();
+
+            result.ShouldBeEmpty();
+        }
+
+        [Fact]
+        public void ShouldReturnMultipleLinqExpressions()
```

Comment:
What if only one value starts with `linq.` and another is in the wrong format? We could use a test like that.

Reply:
added ShouldOnlyValidateCorrectFormat test